wordpress-mobile/gutenberg-mobile
# Update React Native Aztec version.

This PR updates the version of Aztec being used in the project.

This new version fixes an issue where changes of style/format where done on the content of a block and where not being propagated to the HTML.

How to test:
 - Start the demo app
 - Select text on a paragraph block
 - Tap on the Bold button on the style toolbar
 - Switch to HTML
 - Check the the `strong` tag was applied.
